docs/proposals/scc-check.md

```diff
@@ -0,0 +1,89 @@
+## Problem
+It is impossible for a user to know whether the `PodSpec` he's describing will actually be allowed by the current SCC rules.
+This becomes even more problematic when trying to reason about whether a `DeploymentConfig` or `ReplicationController` or other resource containing a `PodSpec` 
+that a system component will eventually create will work.
```

Comment:
Let's call it PodSecurityPolicyReview then.

On Tue, May 24, 2016 at 8:22 AM, David Eads <notifications@github.com>
wrote:

> In docs/proposals/scc-check.md
> <https://github.com/openshift/origin/pull/8199#discussion_r64380464>:
>
> > @@ -0,0 +1,89 @@
> > +## Problem
> > +It is impossible for a user to know whether the `PodSpec` he's describing will actually be allowed by the current SCC rules.
> > +This becomes even more problematic when trying to reason about whether a `DeploymentConfig` or `ReplicationController` or other resource containing a `PodSpec`
> > +that a system component will eventually create will work.
>
> In the future when we have dry-run of admission is this obviated?
>
> Yes, but the problem is so annoying and intractable now, I think its worth
> the API. I don't think that we'll get a dry-run of admission as quickly as
> we can get this API.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8199/files/ff585daf7fe59eea1ebd4c0cb46386ab656d41ca#r64380464>
>